internal/apigw/apigw_openapi_schema.go

```diff
@@ -86,8 +100,23 @@ func (sc *schemaContainer) Message(m pgs.Message, filter []string, nullable *boo
 				continue
 			}
 		}
+		mext := &apigw_v1.FieldOptions{}
+		_, err := f.Extension(apigw_v1.E_Field, mext)
+		if err != nil {
+			return nil
+		}
+		if len(mext.FieldOptions) > 0 {
+			fOption := mext.FieldOptions[0]
+			if fOption.Required {
+				required = append(required, jn)
+			}
+		}
```

Comment:
Might be valuable to have a `getFieldOptions` just like `getMessageOptions`

Reply:
added in most recent commit